powerpacks.go

```diff
@@ -267,20 +308,33 @@ func (service *PowerpackService) Count_numbers(params PowerpackSearchParam) (cou
 }
 
 func (service *PowerpackService) Find_numbers(number string) (response *NumberResponse, err error) {
+	params := FindNumberOptions{}
+	response, err = service.FindNumbersWithOptions(number, params)
```

Comment:
Does it default to SMS when empty? If yes, this should be documented in the structs.